### ISSUE TITLE
New room list: Accessibility issue: Icons are too large

### ISSUE THREAD

**Issue by Felthry** [vector-im/riot-web#14489]
### Description

The room list seems to have much larger icons, making it take up far more room than usual. Since I like to keep riot/element taking up about a quarter of my screen, rather than the whole screen, this means that it only takes about ten people or rooms to fill the list to a point that it has to scroll, where previously it took more like 17-20.

I don't have any screenshots because 1) I didn't take any before to show before and after shots and 2) I can't take screenshots without compromising privacy, and pixellating it would make it hard to see what the complaint is about.

Note that while I consider this an accessibility issue, having the icons too *small* could also be an accessibility issue. I would say it should be toggleable, at least between the old size and the current size if not more sizes.

### Version information

- **Platform**: desktop

For the desktop app:

- **OS**: Windows 10 Education
- **Version**: 1.7.0

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/riot-web/issues/14099
